code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -107,7 +107,7 @@
 
 	//Removing from inventory
 	if(href_list["remove_inv"])
-		if(!Adjacent(usr) || !(ishuman(usr) || ismonkey(usr) || isrobot(usr) ||  isalienadult(usr)))
+		if(!Adjacent(usr) || !(ishuman(usr) || issmall(usr) || isrobot(usr) ||  isalienadult(usr)))

```

Comment:
as far as I can tell, all of those are encaptured in an !(), and monkies will succeed the ishuman part, so it will return regardless of them being small or not.